src/utils/normalize-mime-parts.ts

```diff
@@ -97,8 +101,31 @@ export function normalizeMimeParts(
 			if (disposition !== 'attachment') {
 				let bodyType =
 					type === 'text/html' ? 'html' : type === 'text/plain' && 'text';
-				if (bodyType && (!acc[bodyType] || disposition !== 'inline')) {
-					acc[bodyType] = content;
+
+				if (
```

Comment:
would that be a good idea to explain this flow in a comment? So that anyone who reads the code knows why this special case was written.

Reply:
Done